docs/index.md

```diff
@@ -65,76 +69,76 @@ POST {json file} <url>/insert/
+    {"lab_id": 1234567891011121, "type": "BLL", "date": "2012-03-20", "sample_type": "V", "result": "13.6"}, 
+    {"lab_id": 1234567891011121, "type": "BLL", "date": "2012-02-21", "sample_type": "V", "result": "15"}, 
+    {"lab_id": 1234567891011121, "type": "BLL", "date": "2011-08-19", "sample_type": "V", "result": "5"}] 
 }
 ```
 
 ### Response
 
 #### Body
 
-| Field            | Format       | Constraints                                                                     | Notes/Questions                                                                                  |
-|------------------|--------------|---------------------------------------------------------------------------------|--------------------------------------------------------------------------------------------------|
-| version          | Text         | "0.3.0"                                                                         | Follows Semantic Versioning 2.0.0 http://semver.org/spec/v2.0.0.html                             |
-| timestamp        | TimeDate     | yyyy-mm-ddThh:mm:ss.sss-hh:mm                                                   | Current Timestamp. RFC 3339 (ISO 8601 derivative).                                               |
-| visit_date       | Date         | yyyy-mm-dd                                                                      | Effective date of score                                                                          |
-| alliance_id      | Text         | AlphaNumeric                                                                    | Return value provided by Feed                                                                    |
-| risk_score       | AlphaNumeric | 9.99                                                                            | Currently expecting numeric, future may be phrased by stating an odds ratio with the risk score. |
-| risk_score_notes | Text         | Additional notes (referral, remediation funds, etc to be returned to provider ) |                                                                                                  |
+| Field            | Format       | Constraints                       | Notes/Questions                                                                                  |
+|------------------|--------------|-----------------------------------|--------------------------------------------------------------------------------------------------|
+| version          | Text         | "0.3.0"                           | Follows Semantic Versioning 2.0.0 http://semver.org/spec/v2.0.0.html                             |
+| timestamp        | TimeDate     | yyyy-mm-dd hh:mm:ss.sss-hh:mm     | Current Timestamp (RFC 3339 complient date).                                                     |
+| visit_date       | Date         | yyyy-mm-dd                        | Effective date of score                                                                          |
```

Comment:
If `visit_date` is removed per my suggestion above, it should also be removed here.

Reply:
Makes sense -- also removing this.